tests/integration/questionnaire/test_questionnaire_progress_value_source_in_repeating_sections.py

```diff
@@ -0,0 +1,259 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnaireProgressValueSourceInRepeatingSections(IntegrationTestCase):
```

Comment:
Applies to other files also, it would help to deduplicate these. It would also help with readability. There are a lot of coomon statements. I suggest doing it for any duplicate groupings, i.e answer same question, asserting section status etc.

Reply:
Added a few methods for each of these progress integration tests. Lmk if that's enough